### DISCUSSION TITLE
Apply a master CSS stylesheet to zim pages via a configuration file?

### DISCUSSION THREAD

**Discussion by kyrathasoft** [zim-desktop-wiki/zim-desktop-wiki#2124] [Q&A] [answered]
Is there a way to apply a custom master CSS style to an entire notebook? If so, how?

If not, is there a way (perhaps a plugin) to apply a master style sheet when exporting a notebook to HTML?

**Comment by introt** [comment, marked as answer]
Please see "GUI style config file" under Help:Config Files in the manual. You'll find answers to both of your questions there.

**Comment by kyrathasoft** [comment]
Thanks!

On Sat, Aug 13, 2022 at 12:24 PM introt ***@***.***> wrote:

> Please see "GUI style config file" under Help:Config Files in the manual.
> You'll find answers to both of your questions there.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/2124#discussioncomment-3390152>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB3FUIGFVOMBBZXUGXEMRWLVY7D3HANCNFSM56OLKVKA>
> .
> You are receiving this because you authored the thread.Message ID:
> <zim-desktop-wiki/zim-desktop-wiki/repo-discussions/2124/comments/3390152@
> github.com>
>
-- 
“To all of us, who have absolutely no control over our future. May we love
each other and share laughter until the end, whenever and however it might
come."

**Reply by kyrathasoft** [reply]
That did the trick, allowing me to set preferred font, etc. If the TextView background color editable?

**Reply by introt** [reply]
Please see https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/1802#discussioncomment-1798274
